File: src/main/java/no/nav/domain/pensjon/regler/kode/VilkarVurderingCti.java
package no.nav.domain.pensjon.regler.kode;

/**
 * Beskriver hvilken vurdering saksbehandler har lagt til grunn for resultatvurderingen.
 * Se K_VILKAR_VURD_T.
 */
public class VilkarVurderingCti extends TypeCti {

    

    public VilkarVurderingCti() {
        super();
    }

    public VilkarVurderingCti(String kode) {
        super(kode);
    }

    public VilkarVurderingCti(VilkarVurderingCti vilkarVurderingCti) {
        super(vilkarVurderingCti);
    }
}

File: src/main/kotlin/no/nav/pensjon/regler/domain/kode/VilkarVurderingCti.kt
package no.nav.pensjon.regler.domain.kode

/**
 * Beskriver hvilken vurdering saksbehandler har lagt til grunn for resultatvurderingen.
 * Se K_VILKAR_VURD_T.
 */
class VilkarVurderingCti : TypeCti {
    constructor() : super()
    constructor(kode: String?) : super(kode)
    constructor(vilkarVurderingCti: VilkarVurderingCti) : super(vilkarVurderingCti)
}